app-frontend/src/app/core/core.module.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+import angular from 'angular';

```

Comment:
Some of this stuff in `core/` looks left over from the generator and should be deleted.